### PR TITLE
feat: send detail of quiz answer to admin

### PR DESCRIPTION
**Pull request by Moleus**
fixes: #50 

Add additional fields to quiz answer sent to admin: `(quiz-name, student-name, group, student-tg-id)` . Updated tests accordingly.

Clojure is the toughest language I've met so far... :joy:

### CODE REVIEW

File: src/course_bot/quiz.clj [hunk 1]
```diff
@@ -30,7 +30,7 @@
     :what-question-yes-no "What (yes/no)?"
     :remember-your-answer "Remember your answer: "
     :quiz-passed "Thanks, quiz passed. The results will be sent when the quiz is closed."
-    :quiz-answers "Quiz answers: "
+    :quiz-answers "Quiz answers: %s (%s, %s, %s, %s)"
     :incorrect-answer "I don't understand you, send the correct answer number (1, 2...)."}}
```
Comment (ryukzak) [inline src/course_bot/quiz.clj:34]: Naming conventions-5 to the end

File: src/course_bot/quiz.clj [hunk 2]
```diff
@@ -296,7 +296,9 @@
                (let [{quiz-key :key quiz :quiz} (current-quiz! tx conf)
                      question-index (count results)
                      next-question-index (+ 1 question-index)
-                     new-results (concat results (list text))]
+                     new-results (concat results (list text))
+                     {student-name :name group :group} (codax/get-at tx [id])
```
Comment (ryukzak) [inline src/course_bot/quiz.clj:300]: Avoid db access onveach answer

File: src/course_bot/quiz.clj [hunk 3]
```diff
@@ -296,7 +296,9 @@
...
+                     quiz-name (:name quiz)]
```
Comment (ryukzak) [inline src/course_bot/quiz.clj:301]: Inline

**Review by ryukzak — CHANGES_REQUESTED** [review]

File: src/course_bot/quiz.clj (hunk 1 above)
Comment (Moleus) [reply src/course_bot/quiz.clj:34]: added `-5` suffix

File: src/course_bot/quiz.clj (hunk 2 above)
Comment (Moleus) [reply src/course_bot/quiz.clj]: moved the read from db to the end of the quiz

**Review by ryukzak — APPROVED** [review]

**Comment by ryukzak** [comment]
drop link on this pr to me telegram. I should know who are you

**Pull request merged**